go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/spiffe/spire-controller-manager
 
-go 1.22
+go 1.22.1
```

Comment:
why using 1.22.1? I think we must use 1.22

Reply:
https://github.com/spiffe/spire-controller-manager/pull/330#discussion_r1548528175